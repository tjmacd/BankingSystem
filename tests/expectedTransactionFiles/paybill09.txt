10                      00000 00000000 A 
03 Account 1            00001 00000.90 EC
00                      00000 00000000   