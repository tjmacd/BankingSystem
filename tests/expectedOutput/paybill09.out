Please select session type (standard or admin):
Logged in as admin
Please enter account holder's name:
Enter the account number:
Enter the payee company:
Enter the amount to pay:
$0.90 payed to EC
Logged out